operator/operator-example-standalone.yaml

```
@@ -0,0 +1,166 @@
+# yaml-language-server: $schema=https://api.omnistrate.cloud/2022-09-01-00/schema/service-spec-schema.json
+name: Redis Operator # Service Plan Name
+deployment:
+  hostedDeployment:
+    GcpProjectId: "app-plane-dev-f7a2434f"
+    GcpProjectNumber: "747498641488"
+    GcpServiceAccountEmail: "omni-btstrp-org-q9h5kjuc0n@app-plane-dev-f7a2434f.iam.gserviceaccount.com" 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)